__tests__/app.test.js

```diff
@@ -33,4 +33,40 @@ describe("GET api/topics", () => {
                 });
         });
     });
-});
\ No newline at end of file
+});
+
+describe("GET /api/articles/:article_id", () => (
+    describe("Connecting to path", () => {
+        test("Should return status 200", () => {
+            return request(app).get("/api/articles/1").expect(200)
+        });
+        test("Should return the correct data for specific article (1)", () => {
+            return request(app).get("/api/articles/1").expect(200)
+                .then(({ body: {article} }) => {
+                    expect(article).toMatchObject({
+                        article_id: 1,
+                        title: "Living in the shadow of a great man",
+                        topic: "mitch",
+                        author: "butter_bridge",
+                        body: "I find this existence challenging",
+                        created_at: "2020-07-09T20:11:00.000Z",
+                        votes: 100,
+                        article_img_url:
+                          "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
+                      })
```

Comment:
Nice use of toMatchObject here and it's good to see that you have checked the data of the specific article.